### DISCUSSION TITLE
Crashing after May 1st update?

### DISCUSSION THREAD

**Discussion by tylermaillet** [akiver/cs-demo-manager#822] [General]
Hey,
I'm not sure if it's just me, since I haven't seen any posts about it yet, but the game is immediately crashing after loading into the demo whenever I try to play it from the demo manager. It doesn't let me play the demo using one of the steam beta versions. I know the new update changed some stuff with demos so it makes sense why it would be crashing, but I just haven't heard anything since the update came out and figured I'd make a comment here to see if it was something on my end.

**Comment by tju-simple** [comment]
yeah my game crashed too. demo from perfectworld

**Comment by tmehaffy** [comment]
My game crashes when trying to view the demo in any way through demo manager. Really wanting to watch highlights with my teamates again.

**Comment by N1N52** [comment]
Same problem here, game crashes trying to play demos from any sources, faceit, valve... older demos and recent demos...

**Comment by akiver** [comment]
dup #818
